buildenv/jenkins/JenkinsfileBase

```diff
@@ -375,13 +375,15 @@ def runTest( ) {
 		timestamps{
 			echo 'Running tests...'
 			def CUSTOM_OPTION = ''
+			int iterationCount = 1
 			
 			TARGET = "${params.TARGET}";
 			if (TARGET.contains('custom') && CUSTOM_TARGET!='') {
 				CUSTOM_OPTION = "${TARGET.toUpperCase()}_TARGET='${CUSTOM_TARGET}'"
 			}
 			RUNTEST_CMD = "./openjdk-tests _${params.TARGET} ${CUSTOM_OPTION}"
 			for (int i = 0; i < ITERATIONS; i++) {
+				echo "ITERATION: ${iterationCount}"
```

Comment:
Your PR looks good @M-Davies - one question, did you want to include out of what total ? echo "ITERATION: ${iterationCount}/ITERATIONS"

Reply:
Yes that would be useful :)